#!/bin/sh -e
## 01_aclocal_underquote_warning.dpatch by Jordi Mallach <jordi@debian.org>
##
## DP: Fix automake 1.8 warning by doing proper quoting.

if [ $# -lt 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0

@DPATCH@
diff -urNad /home/jordi/deb/alsa/alsa-lib/utils/alsa.m4 alsa-lib/utils/alsa.m4
--- /home/jordi/deb/alsa/alsa-lib/utils/alsa.m4	2003-09-09 03:08:38.000000000 +0200
+++ alsa-lib/utils/alsa.m4	2004-01-20 13:25:27.000000000 +0100
@@ -12,7 +12,7 @@
 dnl For backwards compatibility, if ACTION_IF_NOT_FOUND is not specified,
 dnl and the alsa libraries are not found, a fatal AC_MSG_ERROR() will result.
 dnl
-AC_DEFUN(AM_PATH_ALSA,
+AC_DEFUN([AM_PATH_ALSA],
 [dnl Save the original CFLAGS, LDFLAGS, and LIBS
 alsa_save_CFLAGS="$CFLAGS"
 alsa_save_LDFLAGS="$LDFLAGS"
